src/ucp/core/ucp_context.h

```diff
@@ -35,9 +35,14 @@ typedef struct ucp_context_config {
     size_t                                 zcopy_thresh;
     /** Estimation of bcopy bandwidth */
     size_t                                 bcopy_bw;
+    /** Segment size in the worker pre-registered memory pool */
+    size_t                                 worker_seg_size;
```

Comment:
just seg_size, since many other configs are also for the worker (e.g zcopy_thresh) but they don't have worker_ prefix in the name